docs/Community.md

```diff
@@ -34,6 +34,8 @@ When you want to show your users and contributors that they can use pixi in your
   - [HydroMT](https://github.com/Deltares/hydromt): Automated and reproducible model building and analysis
   - [HydroMT SFINCS](https://github.com/Deltares/hydromt_sfincs): SFINCS plugin for HydroMT
   - [PyFlwDir](https://github.com/Deltares/pyflwdir): Fast methods to work with hydro- and topography data in pure Python.
+- USGS:
```

Comment:
You're planning to add more? ;P

Reply:
Haha, I expect that they will add more on their own. That's what happened with Deltares at least :P